chart/epinio-installer/README.md

```diff
@@ -13,5 +13,6 @@ The doc is centralized in a uniq place, checkout the [doc website](https://docs.
 
 ## Development
 
-    % helm upgrade --install -n epinio --create-namespace --set "skipLinkerd=true" epinio-installer chart/epinio-installer
+    % helm repo add epinio https://epinio.github.io/helm-charts/
+    % helm upgrade --install -n epinio --create-namespace --set "skipLinkerd=true" epinio-installer epinio/epinio-installer
```

Comment:
This section is titled `Development`. If the instructions are indeed for development, we shouldn't install from the remote repository but rather from the directory. Maybe we should have a separate section for the "production" installation (in other words from the remote repository)?